spec/models/activity_spec.rb

```diff
@@ -129,36 +129,100 @@
 
     context "overall activity state" do
       context "when the activity form is final" do
-        subject { build(:activity, :at_identifier_step, form_state: "complete") }
+        subject { build(:project_activity, :at_identifier_step, form_state: "complete") }
+        it { should be_invalid }
+      end
+    end
+
+    context "when activity is a fund" do
```

Comment:
I'm wondering if these belong in a `spec/validator` test instead 🤔

Reply:
Yeah, probably. That spec is getting pretty big too! 😬